City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-6317: Add missing role delegation permissions for admin users

# [UHF-6317](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6317)

## What was done
* Added missing role delegation permissions for admin users.
  * After this, admin users can give these roles for other users.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git fetch`
  * `git checkout UHF-6317-add-role-delegation-permissions`
  * `make fresh`
* Run `make drush-cr`

## How to test

* Check that an admin user (not uid 1) can add comprehensive_school_editor, daycare_editor and playground_editor roles for other users.

## Designers review
